macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -325,11 +325,31 @@ open class AvatarView: NSView {
 	///
 	/// - note: Internal visibility exists only for unit testing
 	@objc(getColorForIndex:)
+	@available(*, deprecated, message: "Use getInitialsColorSetFromPrimaryText:secondaryText: instead")
```

Comment:
Does this one still need to be public? It seems like it's not used in devmain.

Reply:
Line 360 has a deprecation directive pointing to this one which is also deprecated.

`	@available(*, deprecated, message: "Use getColorForIndex instead")`

Reply:
thanks for catching this, updated the comment

Reply:
The documentation comment seems a bit off too:
"Get the **color** associated with a given index"

it's actually returning a ColorSet object. Can you improve the comments there?

Also, what's the internal visibility comment about?
"- note: Internal visibility exists only for unit testing"

Reply:
I have updated the comment. The note is no longer valid because I changed the test case in previous iteration.

![Screen Shot 2021-04-01 at 8 44 26 AM](https://user-images.githubusercontent.com/63682282/113319634-80fa4f80-92c6-11eb-89c2-5bdd5715577a.png)
 
https://github.com/microsoft/fluentui-apple/pull/455/files#diff-48e9b1830c83b38133ae54b9dca476f4ab456571c9d162135bd0ac6e1ff65c8f